src/devtools/controller.ts

```diff
@@ -57,3 +62,26 @@ export function setOverrides(data: Omit<SetOverridesMessage, "type">) {
     ...data,
   });
 }
+
+if (!isProd) {
+  const mockData: RefreshMessage = {
+    type: "GB_REFRESH",
+    features: {
+      isCoolFeatureEnabled: {
+        defaultValue: false,
+        rules: [],
+      },
+    },
+    experiments: {},
+    attributes: {
+      test: 123,
+    },
+    overrides: {},
+  };
```

Comment:
I wouldn't use mock data in the real app. This is why something like Storybook comes in handy. When a component is structured using presentational-container component architecture, and the features are passed into the main component, e.g. `<DevTools />` (which I think is just `<App />` right now, and probably needs to change), then we can mock it as needed.